db/migrate/20160224155909_add_registered_at_to_subscription_facet.rb

```diff
@@ -0,0 +1,9 @@
+class AddRegisteredAtToSubscriptionFacet < ActiveRecord::Migration
+  def up
+    add_column(:katello_subscription_facets, :registered_at, :datetime, :null => false, :default => Time.now)
```

Comment:
The value for `registered_at` inserted by this migration is meaningless.  The correct value is stored in CP and what is stored in this table is just for convenience (i.e. not having to make a call to CP to get this value).